ACE/ace/Dev_Poll_Reactor.cpp

```diff
@@ -1122,10 +1122,10 @@ ACE_Dev_Poll_Reactor::dispatch_io_event (Token_Guard &guard)
 
   // Define bits to check for while dispatching.
 #if defined (ACE_HAS_EVENT_POLL)
-  const __uint32_t out_event = EPOLLOUT;
```

Comment:
ACE isn't guaranteed to have `uint32_t` and friends on all platforms. Instead use `ACE_UINT32`.

Reply:
I'm little skeptical if this makes sense. On one hand the type I'm suggesting to be replaced was already non ACE_UINT32. On the other hand `#if defined (ACE_HAS_EVENT_POLL)` will never be true for any OS but Linux, since event poll is Linux specific (not part of POSIX).

Reply:
After all we want to match the datatype of the system interface and not just define a 32bit integer.

Reply:
I still don't consider it a good idea, but I've changed it anyway.

Reply:
I think very early versions of poll used __uint32_t, newer versions use uint32_t

Reply:
Yea, I think I said somewhere else if it sensitive about the type, then I was less concerned about making it "proper" ACE code. If it's not right to `ACE_UINT32` then we don't have to use it.